### PR TITLE
Observability build

### PR DESCRIPTION
**Pull request by mtruj013**
## Done
Building /observability/what-is-observability per copydoc: https://docs.google.com/document/d/1eoqDQzFiRJY2u_EO4JBuLBLqae4K3nrEf3qd88KHw0Y/edit

zeplin:
https://app.zeplin.io/project/6143075beaee62250b828788/screen/614307aa6a68af5099c01a44 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-10439.demos.haus/observability/what-is-observability
- check against copy and zeplin mock-up (see above)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4380

## Screenshots
Screenshot 2021-09-21 at 18 03 39

Screenshot 2021-09-21 at 18 03 59
Screenshot 2021-09-21 at 18 04 13

### CODE REVIEW

**Comment by mmanciop** [comment]
@pmahnke when this page goes live we need to fix the footer :-)
Now that the /observability section has more than one page, it can take its rightful place in the footer as section, I'd say above Appliances. WDYT?

File: templates/observability/what-is-observability.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+<section class="p-strip--light">
+    <div class="row">
+        <div class="col-8">
+            <h2>Telemetry correlation for deeper insights</h2>
+            <h3>Making the most out of your metrics, logs, distributed traces and more</h3>
+            <p>The various telemetry types provide you with more insight when they are correlated with one another.</p>
+        </div>
+        <div class="col-12">
+            {{ image (
+                url="https://assets.ubuntu.com/v1/700713ff-telemetry-types.svg",
```
Comment (mmanciop) [inline templates/observability/what-is-observability.html:93]: https://assets.ubuntu.com/v1/1101e891-telemetry_types.svg

File: templates/observability/what-is-observability.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+{% block title %}What is observability{% endblock %}
+
+{% block meta_description %}Observability is fundamental for applications and systems. Learn how it relates with monitoring and the different types of telemetry that is collected and analyzed in the state of the art of open source observability.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1eoqDQzFiRJY2u_EO4JBuLBLqae4K3nrEf3qd88KHw0Y/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip u-no-padding--top u-no-padding--bottom">
+    <div class="p-strip--suru-bottomed">
```
Comment (mmanciop) [inline templates/observability/what-is-observability.html:12]: I like it *much* more with `p-strip--suru-topped` instead :-)

**Review by mmanciop — CHANGES_REQUESTED** [review]
A couple minor changes are needed IMO (see comments), and there is additionally the need to change the footer, but as a first iteration, we are pretty much there :-)

**Comment by mtruj013** [comment]
@mmanciop about to push those changes and open for review now,  but also what do you think about the colon here? I think a period would be more grammatically correct

> Observability is a crucial property of well-designed software systems: your system
> is observable if you can know whether it is working as desired and, if it is not, who
> is impacted and how badly, what the issues are and where they are originating
> from.

**Comment by mmanciop** [comment]
> @mmanciop about to push those changes and open for review now, but also what do you think about the colon here? I think a period would be more grammatically correct
> 
> > Observability is a crucial property of well-designed software systems: your system
> > is observable if you can know whether it is working as desired and, if it is not, who
> > is impacted and how badly, what the issues are and where they are originating
> > from.

I'd be fine with that change, it is easier to read

**Comment by long-chung** [comment]
- [ ] Suru strip in the header is in the wrong place
- [ ] Cross check spacing in the "Monitoring and telemetry types" Hard to see which sub head belongs to copy as spacing looks equal between paragraphs.
- [ ] Check the image ratio on "Telemetry correlation for deeper insights" section. Image looks like its squished in the x-axis

**Comment by mtruj013** [comment]
Hi @long-chung, regarding the suru strip header, @mmanciop mentioned that he preferred it on the top as opposed to the bottom (see comments above). You guys let me know what the final decision on that is and I'll go ahead and implement

**Comment by long-chung** [comment]
@mtruj013 "mentioned that he preferred it on the top as opposed to the bottom" 
Coolies. Thats fine. Proceed with suru strip at the top then. Thanks

**Comment by mtruj013** [comment]
@long-chung Done! lmk if the spacing/ image look better or if they still need a touch up

File: templates/observability/what-is-observability.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="p-strip--light">
+    <div class="row">
+        <div class="col-8">
+            <h2>Observability versus monitoring</h2>
+            <h3 class="p-heading--4">Rather, observability <em>through</em> monitoring</h3>
```
Comment (bethcollins92) [inline templates/observability/what-is-observability.html:37]: not your fault but could you remove the italics please? Conversation [here](https://chat.canonical.com/canonical/pl/bqqyrj7b3ibwifat4eefit3zoh)

File: templates/observability/what-is-observability.html [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+<section class="p-strip--light">
+    <div class="row">
+        <div class="col-8">
+            <h2>Telemetry correlation for deeper insights</h2>
+            <h3 class="p-heading--4">Making the most out of your metrics, logs, distributed traces and more</h3>
+            <p>The various telemetry types provide you with more insight when they are correlated with one another.</p>
+            <p>
+                {{ 
+                    image (
+                    url="https://assets.ubuntu.com/v1/1101e891-telemetry_types.svg",
```
Comment (bethcollins92) [inline templates/observability/what-is-observability.html:98]: I'd hide this image on small screens it's hard to see what's going on on mobile

**Comment by long-chung** [comment]
@mtruj013 Thanks for sorting out the spacing.

I'm still seeing stretching on the Telemetry image. If I view source, its currently rendering at 682x665px, Whereas the original source file is 683x595, so it's getting squashed by 70px on the horizontal axis.

**Review by bethcollins92 — APPROVED** [review]
Nice, LGTM - I'm sure it will look a lot better once all the illustrations are done and can be added

**Pull request merged**